### ISSUE TITLE
Primary button scheme is not focusable

### ISSUE THREAD

**Issue by jonrohan** [primer/view_components#1551]
https://github.com/github/github/pull/244059#discussion_r1007436087

The primary scheme on the Beta button doesn't have a focus ring
